templates/emails/onboarding.day7.hbs

```diff
@@ -1,16 +1,21 @@
-Subject: Give visibility to your backers and sponsors
+Subject: Engage your community on Open Collective
 
 {{> header}}
 
 <h1>Hey {{collective.name}} 👋</h1>
 
-<p>Did you know that we offer widgets to automatically show your backers and sponsors on your website? It's a great way to thank them for their financial contribution. It will also incentivize others to contribute as well!</p>
+<p>Did you know we offer some neat features to better engage with and show gratitude to your community?</p>
 
-<p>Go to the settings of your collective in the "export" tab: <a href="https://opencollective.com/{{collective.slug}}/edit/export">https://opencollective.com/{{collective.slug}}/edit/export</a></p>
+<ul>
+  <li> <a href="https://docs.opencollective.com/help/collectives/communication">Send Updates</a>: Let your backers know what their support is enabling. Updates go out as an email and also automatically show on your Collective page.</li>
```

Comment:
Shouldn't we say `financial contributors`?

```suggestion
  <li> <a href="https://docs.opencollective.com/help/collectives/communication">Send Updates</a>: Let your financial contributors know what their support is enabling. Updates go out as an email and also automatically show on your Collective page.</li>
```

Also I'm not familiar with that sentence form, `their support is enabling` => enabling what? Is it a way to say that it enables the collective to do more things?